Controllers/UserController.cs

```diff
@@ -109,6 +108,67 @@ public IHttpActionResult GetAll()
             return Ok(users);
         }
         //------------------------------------------------------------------------------
+        [AllowAnonymous]
+        [HttpPost]
+        [Route("auth")]
+        public IHttpActionResult Authenticate(LoginRequest loginRequest)
+        {
+            if (loginRequest == null) return BadRequest("Complete the fields to login an user.");
+            User user = new User();
+            try
+            {
+                using (sqlConnection)
+                {
+                    SqlCommand sqlCommand = new SqlCommand(@"SELECT Use_ID
+                                                                 ,Use_Name
+                                                                 ,Use_LastName
+                                                                 ,Use_Identification
+                                                                 ,Use_Password
+                                                                 ,Use_Email
+                                                                 ,Use_Status
+                                                                 ,Use_BirthDate
+                                                                 ,Use_Phone
+                                                             FROM [User]
+                                                             WHERE 
+                                                                Use_Name = @Use_Name 
+                                                                AND Use_Password = @Use_Password", sqlConnection);
+
+                    sqlCommand.Parameters.AddWithValue("@Use_Name", loginRequest.Username);
+                    sqlCommand.Parameters.AddWithValue("@Use_Password", loginRequest.Password);
+
+                    sqlConnection.Open();
+
+                    SqlDataReader sqlDataReader = sqlCommand.ExecuteReader();
+
+                    if (sqlDataReader.Read())
+                    {
+                        user.Use_ID = sqlDataReader.GetInt32(0);
+                        user.Use_Name = sqlDataReader.GetString(1);
+                        user.Use_LastName = sqlDataReader.GetString(2);
+                        user.Use_Identification = sqlDataReader.GetString(3);
+                        user.Use_Password = sqlDataReader.GetString(4);
+                        user.Use_Email = sqlDataReader.GetString(5);
+                        user.Use_Status = sqlDataReader.GetString(6);
+                        user.Use_BirthDate = sqlDataReader.GetDateTime(7);
+                        user.Use_Phone = sqlDataReader.GetString(8);
+
+                        var token = TokenGenerator.GenerateTokenJwt(loginRequest.Username);
+                        user.Token = token;
+                    }
+                    sqlConnection.Close();
```

Comment:
when a SQL connection is open inside a using() the connection is closed when the execution go out from the using() so is not necessary call a Close() inside a using()

Reference: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement